src/engine/source/kvdb2/src/kvdbSpace.cpp

```diff
@@ -54,8 +62,28 @@ std::variant<std::string, base::Error> KVDBSpace::get(const std::string& key)
         return value;
     }
 
-    return base::Error { fmt::format("Cannot get key '{}'. Error: {}", value, key, status.getState()) };
+    return base::Error {fmt::format("Cannot get key '{}'. Error: {}", value, key, status.getState())};
 }
 
+std::variant<std::unordered_map<std::string, std::string>, base::Error> KVDBSpace::dump()
+{
+    std::unordered_map<std::string, std::string> content {};
+    std::shared_ptr<rocksdb::Iterator> iter(m_pRocksDB->NewIterator(rocksdb::ReadOptions(), m_pCFhandle));
+
+    for (iter->SeekToFirst(); iter->Valid(); iter->Next())
+    {
+        content[iter->key().ToString()] = iter->value().ToString();
+    }
+
+    if (!iter->status().ok())
+    {
+        return base::Error {
+            fmt::format("Database '{}': Couldn't iterate over database: '{}'", m_dbName, iter->status().ToString())};
```

Comment:
Please, try to avoid shortening the messages. It is preferred to use `could not` over `couldn't`